draft-ietf-rtcweb-jsep.xml

```diff
@@ -3642,27 +3642,34 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         processing MUST stop and an error MUST be returned.
         <list style="symbols">
 
-          <t>If the PeerConnection state is "have-local-offer", and the
-          ICE ufrag or password changed from the previous remote
+          <t>If the ICE ufrag or password are unchanged from the previous remote description,
+          and the description is of type "answer" or "pranswer", and
+          the current local description is an ICE restart, then
+          generate an error.</t>
+
+          <t>If the ICE ufrag or password changed from the previous remote
```

Comment:
This text is somewhat confusing in the case where the description is an answer - it's not clear what "an ICE restart is needed" means, if the offer the client sent already asked for an ICE restart.

Maybe just say "If the ICE ufrag or password changed from the previous remote description:", and then include the 9.1.1.1 reference in the 'offer' bullet.